adapters/quick/src/main/scala/caliban/quick/package.scala

```diff
@@ -0,0 +1,16 @@
+package caliban
+
+import zio.{ RIO, Trace, ZIO }
+import zio.http.{ RequestHandler, Response }
+
+package object quick {
+
+  implicit class GraphqlServerOps[R](val gql: GraphQL[R]) extends AnyVal {
+    def runServer(port: Int, api: String, graphiql: Option[String] = None)(implicit trace: Trace): RIO[R, Nothing] =
```

Comment:
Can you add scaladocs? Not sure what `api` and `graphiql` are looking at types (are these paths? maybe it should be included in the names)